compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -6,8 +6,8 @@ use rustc_middle::span_bug;
 use rustc_middle::ty::{self, Ty, TyCtxt};
 use rustc_span::Span;
 
-impl<'tcx> crate::MirBorrowckCtxt<'_, '_, '_, 'tcx> {
-    pub fn dcx(&self) -> DiagCtxtHandle<'tcx> {
```

Comment:
Would `'infcx` be a better lifetime name than `'cx` for this second commit?

I see now that `infcx` is used for both `InferCtxt` (the common case) and `BorrowckInferCtxt` (the less common case, but the relevant one here). I wonder if `binfcx` would be a better name to make that distinction clearer.